sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/query.sql

```diff
@@ -54,6 +54,16 @@ activations AS (
     client_id
 ),
 -- Find earliest data per client from the first_session ping.
+first_session_ping_min_seq AS (
+  SELECT
+    client_info.client_id AS client_id,
```

Comment:
This would be required per sample_id to match the first_session_query?

Reply:
Can you elaborate on this?  Thanks!